myuw/data/forward_mapping.csv

```diff
@@ -0,0 +1,19 @@
+domain,login_url,title,icon_style
+deskmail.washington.edu,https://alpine.washington.edu,UW Deskmail,fa-envelope
+gamail.uw.edu,https://mail.google.com,UW Gmail,fa-google
+exchange.washington.edu,https://outlook.office365.com,UW Office 365,fa-windows
+gmail.com,https://mail.google.com,GMail,fa-google
+hotmail.com,https://mail.live.com/m,Hotmail,fa-windows
+yahoo.com,https://mail.yahoo.com,Yahoo,fa-yahoo
+comcast.net,https://login.comcast.net,Comcast,fa-envelope
+msn.com,https://www.msn.com,MSN,fa-windows
+aol.com,https://my.screenname.aol.com,AOL,fa-envelope
+live.com,https://mail.live.com/m,MS Live,fa-envelope
```

Comment:
These two works equally well for me:
https://outlook.live.com/owa/
https://mail.live.com/